portlist/poller.go

```diff
@@ -45,6 +45,10 @@ type Poller struct {
 	scratch []Port
 
 	prev List // most recent data, not aliasing scratch
+
+	// Include updates for loopback addresses.
+	// This field should only be changed before calling Run.
+	IncludeLocalhost bool
```

Comment:
put the knobs at the top of the struct.

also, use conventional go doc style: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

// IncludeLocalhost controls whether services bound to localhost are included.
//
// This field should only be set before calling Run.